tools/build.hpp

```diff
@@ -18,7 +18,7 @@ namespace antler {
          subcommand->footer(std::string(R"(Examples:)")
                + "\n\t" + app.get_name() +R"( build -j3)");
          subcommand->add_option("-p, path", path, "This is the path to the root of the project.");
-         subcommand->add_option("-j, --jobs", jobs, "The number of submodules fetched at the same time.")->default_val(1);
+         subcommand->add_option("-j, --jobs", jobs, "The number of submodules fetched at the same time. Default is 0, which means to use git default.")->default_val(0);

```

Comment:
Maybe copy the text from `git --help` and specify what git will do when passed 0:
```suggestion
         subcommand->add_option("-j, --jobs", jobs, "The number of submodules cloned in parallel. Default is 0, use cpu count.")->default_val(0);
```

Reply:
I don't like that because of if `git` going to ever change it we should do the same. Also I'm not sure this is same for all git versions so I would rather leave that to git. Your statement sounds like more specific but not 100% correct in all cases. my git says:
`       -j <n>, --jobs <n>
           The number of submodules fetched at the same time. Defaults to the submodule.fetchJobs option.
`

Reply:
Fair point.But I'm not sure sending `-j0` uses git's default either.

After more investigation, I don't think `--jobs` is used _unless_ `--recurse-submodules` is also specified. We don't do that. I am less certain that this is the right answer for #28.

Reply:
maybe let's just remove it at all? Any clue why it was added? It seems to me this is useless. At least in mvp.

Reply:
No clue. I believe removal is appropriate. Our code will be simpler and more maintainable without jobs.

Reply:
So I removed its usage for `clone` however it was also used for `cmake` build so I set it to processors number by default if specifying zero